app/views/account/SubscriptionView.coffee

```diff
@@ -281,11 +281,7 @@ class PersonalSub
         delete @state
         render()
 
-      else
-        delete @state
-        render()
-
-    else if payPalInfo?.billingAgreementID
+    if not @subscribed and payPalInfo?.billingAgreementID
```

Comment:
What's the point of this change?

Reply:
Allows a previously subscribed stripe user to function with a PayPal sub later.  Unlikely to be a problem in the wild, but hampered testing locally.